modules/connectivity/ParticipantConnectionStatus.js

```diff
@@ -195,7 +195,8 @@ ParticipantConnectionStatus.prototype._changeConnectionStatus
             }));
 
         // and analytics
-        Statistics.analytics.sendEvent('peer.conn.status', null, newStatus);
+        Statistics.analytics.sendEvent('peer.conn.status',
+            {value: null, label: newStatus});
```

Comment:
I guess that with the old API we had to specify a `value` of `null` because we didn't care about it but wanted to specify a `label` that we cared about. Why do we specify the `value` with the new API though?